app/controllers/concerns/browse_everythingable.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+module BrowseEverythingable
```

Comment:
Should we just add this to `Valhalla::ResourceController`?

Reply:
Some controllers don't make use of BrowseEverything, `EphemeraFieldsController` for instance, but do include `Valhalla::ResourceController`. If that isn't an issue, then this seems reasonable. Alternately, we could put this into `BaseResourceController`.

Reply:
I'm not sure it's a problem. I wonder if there's some sort of trigger or organization we should be thinking about which is along the lines of "has filesets" and "doesn't have filesets"